Sources/TuistDependencies/DependenciesController.swift

```diff
@@ -65,6 +65,16 @@ public protocol DependenciesControlling {
         swiftVersion: TSCUtility.Version?
     ) throws -> TuistCore.DependenciesGraph
 
+    /// Fetches dependencies.
+    /// - Parameter path: Directory where project's dependencies will be fetched.
+    /// - Parameter packageSettings: Custom Swift Package Manager settings
+    /// - Parameter swiftVersion: The specified version of Swift. If `nil` is passed then the environment’s version will be used.
+    func fetch(
```

Comment:
Only a temporary duplication of `fetch` and `update` methods. This controller should ideally be completely removed with the removal of `Dependencies.swift` for Tuist 4